cmd/collector/app/handler/thrift_span_handler.go

```diff
@@ -121,13 +125,23 @@ func (h *zipkinSpanHandler) SubmitZipkinBatch(spans []*zipkincore.Span, options
 		return nil, err
 	}
 	responses := make([]*zipkincore.Response, len(spans))
-	for i, ok := range bools {
+	// at this point we may have len(spans) < len(bools) if conversion results in more spans
```

Comment:
Perhaps the comment should refer to `total converted` instead of `bool` (same below)?

Reply:
Can you elaborate / suggest?

Reply:
Sorry, please ignore this one, I later understood the context behind `bools`. It's fine as is.